teamscale-client/src/main/java/com/teamscale/client/ITeamscaleService.java

```diff
@@ -80,7 +81,26 @@
 			@Query("end") CommitDescriptor end,
 			@Query("partitions") String partition,
 			@Query("includeNonImpacted") boolean includeNonImpacted,
-			@Body List<ClusteredTestDetails> report
+			@Body List<ClusteredTestDetails> availableTests
+	);
+
+	/** Test Impact API. */
+	@GET("p/{projectName}/test-impact")
+	Call<List<PrioritizableTestCluster>> getImpactedTests(
```

Comment:
The GET endpoint does not return clusters as the information on which tests should be put into which cluster is provided by the PUT request body only.
```suggestion
	Call<List<PrioritizableTest>> getImpactedTests(
```

Reply:
dang, I overlooked that, thanks!